LinuxMac.sh

```diff
@@ -45,10 +45,19 @@ If possible we try to:
 install MambaForge including mamba then Create a mssenv then Install MSS.
 "
 
-CHECKCONDA=$(which conda)
-if [[ $CHECKCONDA == *"miniconda"* ]] || [[ $CHECKCONDA == *"anaconda"* ]]; then
- echo "Found a anaconda/miniconda installation see documentation for a manual installation";
- exit 1;
+sleep 2
+echo "..."
+echo "Checking for a miniconda or anaconda based installation"
+
+if command -v conda  &> /dev/null
+then
+    CHECKCONDA=$(which conda)
```

Comment:
`which conda` gives a complex shell function for me.
The "command" thingie of yours seems way more sensible. Shouldn't we abort in case this gives an output?

Reply:
at the first check it can be also conda from mambaforge. On that we could install MSS. This enables the script to use an existing mambaforge setup. I added for this a workflow CI check, install mambaforge and then MSS.

when it founds anaconda/miniconda there are different oldstates possible and that is probably better manual solved.  There is some risk to break an existing anaconda installation.